zebedee-cms/src/test/java/com/github/onsdigital/zebedee/util/versioning/VersionsServiceImplTest.java

```diff
@@ -0,0 +1,281 @@
+package com.github.onsdigital.zebedee.util.versioning;
+
+import com.github.onsdigital.zebedee.content.page.base.PageDescription;
+import com.github.onsdigital.zebedee.content.page.statistics.dataset.Dataset;
+import com.github.onsdigital.zebedee.content.page.statistics.dataset.Version;
+import com.github.onsdigital.zebedee.content.page.statistics.document.bulletin.Bulletin;
+import com.github.onsdigital.zebedee.exceptions.NotFoundException;
+import com.github.onsdigital.zebedee.model.Collection;
+import com.github.onsdigital.zebedee.reader.CollectionReader;
+import com.github.onsdigital.zebedee.reader.ContentReader;
+import com.github.onsdigital.zebedee.reader.ZebedeeReader;
+import com.github.onsdigital.zebedee.session.model.Session;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.io.File;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.when;
+
+public class VersionsServiceImplTest {
+
+    static final String CONTENT_PATH = "/a/b/c/data.json";
+    static final String CONTENT_URI = "/a/b/c";
+    static final String VERSOIN_URI = CONTENT_URI + "/previous/v1";
```

Comment:
We got a little typo here on the VERSION_URI var name (and throughout this file)